frontend/components/ResourceCreationPage.tsx

```diff
@@ -46,7 +46,7 @@ type ResourceCreationPageProps = {
   years: Year[]
   majors: Major[]
   tags: Tag[]
-  studentTypes: StudentType[]
+  student_types: StudentType[]
```

Comment:
Do we still want to do this with @esinx's refactor? Should rebase on master and fix.

Reply:
we can merge this pr first then fix this in the refactor. I think #418 still needs some testing so I would probably rebase from that branch instead.